addons/knowledge/static/src/js/wysiwyg/knowledge_article_link.js

```diff
@@ -92,7 +92,7 @@ const KnowledgeArticleLinkModal = Dialog.extend({
             formatSelection: (data, container, escapeMarkup) => {
                 const markup = [];
                 if (typeof data.icon !== 'undefined') {
-                    markup.push(`<i class="fa fa-w ${escapeMarkup(data.icon)} mr-1"/>`);
+                    markup.push(escapeMarkup(data.icon) + ' ');
```

Comment:
why ?

Reply:
Because with the emojipicker, the emoji is a text character. In short what i do here is adding a character (emoji) and a space before the title of the article:
`[emoji]_space_[articleTitle]`